builder/actions/cmake.py

```diff
@@ -142,7 +142,15 @@ def _build_project(env, project, cmake_extra, build_tests=False, args_transforme
             if value:
                 compiler_flags.append(
                     '-DCMAKE_{}_COMPILER={}'.format(opt.upper(), value))
-    cmake_args = UniqueList([
+
+    # Removed UniqueList to make an ordinary list instead. Having multiple arguments in different parts of the command
```

Comment:
i guess the down side is that the order added to the list really matters now (Probably also matter before). But, we don't have a clear doc about when the args will be added to the list here, which will make this hard to use.